nireports/reportlets/nuisance.py

```diff
@@ -93,11 +93,11 @@ def plot_dist(
 
     gsp = GridSpec(2, 1)
     ax = fig.add_subplot(gsp[0, 0])
-    sns.distplot(data.astype(np.double), kde=False, bins=100, ax=ax)
+    sns.displot(data.astype(np.double), kde=False, bins=100, ax=ax)
     ax.set_xlabel(xlabel)
 
     ax = fig.add_subplot(gsp[1, 0])
-    sns.distplot(np.array(distribution).astype(np.double), ax=ax)
+    sns.displot(np.array(distribution).astype(np.double), ax=ax)
```

Comment:
Simplifying the `np.array()` call slightly, as well.

```suggestion
    sns.histplot(np.array(distribution, dtype="f8"), kde=True, stat="density", kde_kws={"cut": 3}, ax=ax)
```